utils/GlobalPOM.ts

```diff
@@ -0,0 +1,21 @@
+import { LoginPage } from "./components/loginPage";
```

Comment:
Hmmm, why this file is called GlobalPOM under utils folder?
Usually PO structire is the folllowing:
pages/concretePage.ts